src/Misc.php

```diff
@@ -199,6 +199,9 @@ public static function fix_protocol($url, $http = 1)
         return $url;
     }
 
+    /**
+     * @deprecated since SimplePie 1.7.1, use PHP native array_replace_recursive() instead.
```

Comment:
Maybe also mention that the behaviour is slightly different in case someone really heeds this recommendation?

Reply:
I don't think that's necessary. The developer must always check if and how one can use an offered alternative.